phidgets_motors/src/motors_ros_i.cpp

```diff
@@ -135,7 +138,10 @@ MotorsRosI::MotorsRosI(const rclcpp::NodeOptions& options)
         for (int i = 0; i < n_motors; ++i)
         {
             publishLatestDutyCycle(i);
-            publishLatestBackEMF(i);
+            if (motors_->backEMFSensingSupported(i))
```

Comment:
I think it is better to move this check into `publishLatestBackEMF`; that way, we avoid duplicating the logic.